code/modules/jobs/job_types/roguetown/church/confessor.dm

```diff
@@ -1,59 +1,72 @@
 /datum/job/roguetown/shepherd
-	title = "Confessor"
+	title = "Mook"
 	flag = CONFESSOR
 	department_flag = CHURCHMEN
 	faction = "Station"
-	total_positions = 0
-	spawn_positions = 0
+	total_positions = 3
+	spawn_positions = 3
 
-	allowed_races = RACES_TOLERATED_UP
-	allowed_sexes = list(MALE)
-	tutorial = "Confessors are shady agents of the church hired to spy on the populace and keep them moral. As the most fanatical members of the clergy, their main concern is assisting the local Puritan with their work in extracting confessions of sin as well as hunting night beasts and cultists that hide in plain sight."
+	allowed_races = RACES_VERY_SHUNNED_UP
```

Comment:
I'm not sure if this was an oversight but why can the mooks be literally everything. This is the psydonite church we are talking about. Shouldnt it be RACES_RESPECTED or RACES_TOLERATED_UP at the very least?

Reply:
Mooks are picked from the lowlifes of society. They aren't respected people. Not only to shunned races get a limited amount of roles, but it'd make sense that the lowlife and expendable agents of Psydon be made up of Shunned Races. These men **serve** the Inquisitor, and the Inquisitor should have little respect for them.

Reply:
But if your dumbfuck PR gets merged I'll change it to be more in line with your flaccid lore.

Reply:
I think its best they remain at least, tolerated. Especally with the religious paranoia and fervor inquisitors are apparently known for in lore, they wouldn't want species _spawned from or that probably serve the inhuman from their reputation_, disposable sure but would you trust their _loyalty enough to handle your dirty work under psydon's guidance?_